src/suricata.c

```diff
@@ -257,6 +260,27 @@ void EngineModeSetIDS(void)
     g_engine_mode = ENGINE_MODE_IDS;
 }
 
+void InlineModeSetIDS(void)
```

Comment:
I think this is too broad, lets make something specific for the HTTP body tracking.

Perhaps a setting in the detection engine ctx.